### PR TITLE
pagina 404

### PR DESCRIPTION
**Pull request by nelyflores**

### CODE REVIEW

File: src/pages/404.tsx [hunk 1]
```diff
@@ -1,14 +1,53 @@
...
+          }
+        }
+      }
+    }
+  `);
+  return (
+    <Layout>
+      <SEO title="404: Not found" />
+      <Grid container direction="column" justify="center" alignItems="center">
+        <Img fixed={data.placeholderImage.childImageSharp.fixed} />
```
Comment (the-vladman) [inline src/pages/404.tsx]: why do you add this?

File: src/pages/404.tsx [hunk 2]
```diff
@@ -1,14 +1,53 @@
...
+        <Box m={5} textAlign="center">
+          <Typography variant="h3">Página no encontrada</Typography>
+          <Typography variant="h6">
+            Lo sentimos pero no podemos encontrar la página que buscas
```
Comment (the-vladman) [inline src/pages/404.tsx]: english messages
we will gona use i18n

File: src/pages/404.tsx (hunk 1 above)
Comment (nelyflores) [reply src/pages/404.tsx]: change made

File: src/pages/404.tsx (hunk 2 above)
Comment (nelyflores) [reply src/pages/404.tsx]: change made

**Pull request merged**